src/sdk/pynni/nni/compression/tensorflow/compressor.py

```diff
@@ -75,6 +78,8 @@ class Compressor:
 
     def __init__(self, LayerWrapperClass, model, config_list):
         assert isinstance(model, tf.keras.Model)
+        if isinstance(model, tf.keras.Sequential):
+            raise ValueError('NNI model compression does not support `Sequential` model for now')
```

Comment:
why does not support `Sequential`?

Reply:
Because it's too hard to modify it's attributes properly. `Sequence` abuses complex properties.